js/activity.js

```diff
@@ -491,10 +491,12 @@ class Activity {
                 (event) => {
                     event.preventDefault();
                     event.stopPropagation();
+                    if(this.beginnerMode == false){
```

Comment:
if (this.beginnerMode === false) {

I think we want === instead of == and please add spaces before ( and after ).

Also, the indentation in the following lines needs to be fixed.

Reply:
I think if( !this.beginnerMode ) would be a much better syntax.